salt/tests/unit/modules/files/test_metalk8s_checks.yaml

```diff
@@ -1,3 +1,64 @@
+bootstrap_file:
+  # 1. Success: Nominal
+  - &bstrapf_nominal
+    cp_cidrs: [10.1.0.0/16]
+    wp_cidrs: [10.2.0.0/16]
+    pods_cidr: 10.233.0.0/16
+    svc_cidr: 10.96.0.0/12
+
+  # 2. Fail: pods intersect fully
```

Comment:
Can we have at least one "not obvious" conflict like `10.96.0.0/12` and `10.100.0.0/16` because here we only have conflicts where there are strictly equals